src/hyper_tokio/hyper_ext.rs

```diff
@@ -50,6 +50,14 @@ impl HyperExtensions {
         }
     }
 
+    pub fn token_auth_form_urlencoded(token: Option<&SlackApiToken>) -> String {
```

Comment:
I think this function is too specific and I don't think this will be needed more than in that place, so making this public extension is not necessary.

Also, I would prefer not to have `.unwrap()` code when possible, so I would suggest move this to the calling place as normal code with something like:

```rust
if let Some(token) = maybe_token {
  format!("token={}&", token.value())
}
else {
  String::new()
}
```